code/webpage.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dome Pools, South Africa</title>
+    <style></style>
+    <link rel="stylesheet" href="Style.css" />
+  </head>
+  <body>
+    <h1 class="titles">Dome Pools, Magaliesburg, South Africa</h1>
+    <p class="text">
+      Dome Pools is a wilderness hiking venue, involving an easy walk over a
+      hill and along a stream to some of the biggest and deepest rock pools in
+      the Magaliesberg, making it a fantastic venue for swimming in the summer
+      months.
+    </p>
+    <img
+      src="images\Dome Pools Image 3.jpg"
+      alt="image of waterfall and forest"
+    />
+
+    <h3 class="titles">Why Do I like Dome Pools?</h3>
+    <li>The views</li>
```

Comment:
`<li>` elements must be children of a `<ul>` or `<ol>` in order to be rendered as a list